### ISSUE TITLE
Zoom multiplier on train not working

### ISSUE THREAD

**Issue by MightyBunch** [Fabricators-of-Create/Create#1446]
### Describe the Bug

Zoom multiplier is a config option that change zoom when you are on train in F5. When you on the standing train zoom multiplier working as intended, but when train is moving it's not working and looks like normal F5. I'm not sure is it a speed of train that causing it or turning, but it's happening after a few seconds after train starts.

### Reproduction Steps

1. Sit on train
2. Press F5
3. *train starts*
4. Zoom multiplier is'n working anymore

### Expected Result

Expected that zoom remains after train starts.

### Operating System

Windows 10

### Mod Version

0.5.1f

### Minecraft Version

1.20.1

**Comment by TropheusJ** [comment]
can't reproduce. Please send the game log next time it happens. it's at .minecraft/logs/latest.log.

**Comment by MightyBunch** [comment]
Tested a few things
First of all, nothing appears in the logs.
Next, strangelly, it appears only in world that i played and where discovered that bug. I actually play with a lot of mods, but I tried without any other mods, just create and fabric api, in my world, and bug is still there. But in a new world, same create and fabric api - bug is not happening. Very very strange.
EDIT: Just discovered it in the new world after a 1-2 minutes of circling on train.

**Comment by MightyBunch** [comment]
Video:
https://drive.google.com/file/d/1uK10nE1KfxTyjcf1MmCNWBqDfwHCfVJm/view?usp=sharing

**Comment by SabrCat** [comment]
I have observed this same issue too, I jumped into a train and after a few seconds the camera zoomed back in. For me I was guessing it was because I was playing on a multiplayer server that uses velocity proxy, so I figured a packet may be getting lost. However, this might not have been the case. The only known packet loss happens during login as with the proxy the client is already in the PLAY phase when the server receives the connection, thus forge config lib api port fails to send packets.
However, this doesn't seem applicable here thinking about it again as in my experience all packets sent during runtime arrive without issue.
